examples/model_selection/plot_learning_curve.py

```diff
@@ -70,15 +70,15 @@ def plot_learning_curve(estimator, title, X, y, axes=None, ylim=None, cv=None,
         Refer :ref:`User Guide <cross_validation>` for the various
         cross-validators that can be used here.
 
-    n_jobs : int or None, optional (default=None)
+    n_jobs : int or None, default=None
         Number of jobs to run in parallel.
         ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
-    train_sizes : array-like, shape (n_ticks,), dtype float or int
+    train_sizes : array-like of shape (n_ticks,), dtype=float, int
         Relative or absolute numbers of training examples that will be used to
-        generate the learning curve. If the dtype is float, it is regarded as a
+        generate the learning curve. If the ``dtype`` is float, it is regarded as a
```

Comment:
Line lengths are recommended to be no greater than 79 characters.

Reply:
Ok